### PR TITLE
Make linkify links open in new tab/window

### PR DESCRIPTION
**Pull request by mentos1386**
I believe this behavior is more appropriate.

### CODE REVIEW

**Comment by vincentwoo** [comment]
I think the appropriateness of this flag can vary wildly by setting. Here's a random opinion piece on the topic: https://css-tricks.com/use-target_blank/.

I think setting `target=_blank` by default is fine, but it would be cool if you could toggle it off.

**Comment by parisk** [comment]
Hi @mentos1386, thanks a lot for your contribution! I think it fits the needs of xterm.js a lot, based on its main use cases. @vincentwoo is right though that the appropriateness of this flag can vary and if #198 was closed it would definitely help.

What I would like to see here in order to move with merging is making this an option, by adding an additional argument to the `linkify` and `linkifyTerminalLine` called `target` that will default to `null` and if set it would set the link's `target` attribute to the given value.

At this moment it is important not to break the current behavior of xterm.js.

**Comment by mentos1386** [comment]
I changed it so that you can provide argument and it will set target to what you provide. Default behavior is that it doesn't change anything.

File: addons/linkify/linkify.js [hunk 1]
```diff
@@ -67,6 +69,8 @@
       throw new TypeError(message);
     }
 
+    typeof target === 'undefined' ? target = '' : target = 'target="' + target + '"';
```
Comment (parisk) [inline addons/linkify/linkify.js]: Could you please turn this in the following form to make it more human understandable?

```
if (typeof target === 'undefined') {
  target = '';
} else {
  target = 'target="' + target + '"';
}
```

**Comment by parisk** [comment]
Works great, thanks @mentos1386 🎉 !

**Pull request merged**